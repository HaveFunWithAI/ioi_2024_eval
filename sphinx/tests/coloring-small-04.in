50 500
22 19 14 1 13 5 0 7 0 3 24 10 17 4 12 14 21 8 9 16 15 12 7 16 11 24 3 1 11 6 5 18 23 23 21 18 13 4 22 2 6 10 20 8 19 2 9 17 20 15
0 4
0 16
0 17
0 21
0 23
0 27
0 40
0 47
1 6
1 15
1 20
1 31
1 33
1 34
1 35
1 38
1 47
1 49
2 11
2 19
2 20
2 23
2 28
2 30
2 34
2 36
3 15
3 16
3 20
3 22
3 29
3 33
3 35
3 36
3 39
3 42
3 48
3 49
1 4
2 4
4 10
4 13
4 21
4 28
4 30
4 32
4 35
4 40
4 43
4 44
4 5
5 14
5 24
5 34
5 35
5 42
5 44
5 46
2 6
5 6
6 10
6 11
6 26
6 31
6 33
6 38
6 40
6 42
6 48
2 7
5 7
6 7
7 11
7 14
7 21
7 23
7 25
7 31
7 32
7 34
7 35
7 38
7 43
7 46
7 47
0 8
2 8
4 8
7 8
8 12
8 15
8 32
8 45
0 9
2 9
3 9
5 9
9 12
9 13
9 17
9 23
9 28
9 30
9 31
9 44
9 46
9 48
0 10
2 10
3 10
7 10
10 12
10 17
10 20
1 11
11 20
11 23
11 25
11 32
11 38
11 39
11 45
11 46
11 49
7 12
12 27
12 30
12 31
12 32
12 37
12 41
12 45
12 48
1 13
7 13
8 13
10 13
11 13
13 14
13 23
13 24
13 25
13 28
13 32
13 34
13 35
13 41
13 45
13 46
13 48
13 49
6 14
9 14
14 15
14 18
14 22
14 24
14 29
14 30
14 39
7 15
15 17
15 28
15 32
15 33
15 42
15 45
2 16
12 16
13 16
14 16
16 28
16 29
16 37
16 40
8 17
11 17
13 17
17 22
17 30
17 32
17 33
17 46
17 49
4 18
5 18
15 18
17 18
18 25
18 33
18 35
18 37
18 44
18 47
18 49
1 19
3 19
9 19
11 19
12 19
14 19
16 19
19 22
19 27
19 31
19 39
5 20
9 20
15 20
16 20
19 20
20 25
20 26
20 30
20 36
20 47
5 21
6 21
15 21
17 21
19 21
20 21
21 28
21 34
21 35
21 38
21 39
21 45
21 49
8 22
9 22
10 22
21 22
22 28
22 31
22 35
22 49
17 23
20 23
21 23
23 30
23 31
23 38
23 42
23 44
23 45
7 24
15 24
22 24
24 38
24 41
24 45
1 25
2 25
4 25
6 25
15 25
17 25
19 25
21 25
24 25
25 29
25 41
25 48
0 26
5 26
10 26
21 26
22 26
23 26
25 26
26 28
26 30
26 32
26 39
26 40
8 27
11 27
15 27
17 27
18 27
20 27
21 27
23 27
27 29
27 33
27 38
27 42
27 45
27 47
0 28
1 28
7 28
10 28
18 28
28 35
28 44
1 29
6 29
7 29
15 29
29 30
29 32
29 42
1 30
8 30
10 30
18 30
27 30
28 30
30 31
30 32
30 35
30 39
0 31
3 31
4 31
11 31
14 31
15 31
17 31
20 31
21 31
26 31
27 31
31 36
31 38
31 39
31 44
31 46
1 32
3 32
5 32
10 32
16 32
18 32
20 32
24 32
32 37
32 40
32 44
32 46
4 33
7 33
11 33
14 33
16 33
30 33
33 38
33 43
33 45
0 34
3 34
4 34
8 34
9 34
10 34
14 34
15 34
17 34
34 49
2 35
12 35
16 35
19 35
29 35
33 35
34 35
35 43
1 36
7 36
10 36
13 36
15 36
16 36
17 36
22 36
25 36
33 36
36 42
4 37
8 37
9 37
10 37
14 37
35 37
36 37
37 43
37 44
37 49
3 38
12 38
14 38
26 38
30 38
32 38
34 38
38 39
38 48
38 49
10 39
18 39
20 39
23 39
28 39
39 41
11 40
15 40
18 40
20 40
23 40
36 40
40 44
40 46
40 47
40 48
0 41
8 41
10 41
16 41
20 41
23 41
28 41
37 41
40 41
41 46
8 42
11 42
18 42
20 42
21 42
24 42
25 42
30 42
32 42
33 42
35 42
39 42
40 42
42 44
0 43
5 43
6 43
12 43
13 43
16 43
20 43
28 43
30 43
31 43
36 43
39 43
43 44
11 44
12 44
17 44
20 44
36 44
44 48
2 45
3 45
6 45
29 45
30 45
32 45
41 45
6 46
12 46
19 46
20 46
23 46
26 46
37 46
42 46
43 46
46 47
46 49
13 47
15 47
29 47
30 47
34 47
37 47
39 47
42 47
8 48
10 48
11 48
15 48
23 48
24 48
33 48
34 48
45 48
2 49
30 49
32 49
35 49
36 49
42 49
