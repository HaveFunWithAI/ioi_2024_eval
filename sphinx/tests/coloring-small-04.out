50 404
22 19 14 1 13 5 0 7 0 3 24 10 17 4 12 14 21 8 9 16 15 12 7 16 11 24 3 1 11 6 5 18 23 23 21 18 13 4 22 2 6 10 20 8 19 2 9 17 20 15
